107/src/main/java/org/ehcache/jsr107/ConfigurationMerger.java

```diff
@@ -72,10 +74,10 @@
           builder = templateBuilder;
           log.info("Configuration of cache {} will be supplemented by template {}", cacheName, templateName);
         }
+      } else {
+        builder = handleStoreByValue(jsr107Configuration, builder);
```

Comment:
I agree with what you have done here. If you use a template, you want to benefit from Ehcache specifics without touching JSR-107 config, but we know our _default_ support of _by value_ is inefficient. So indeed, we ignore 107 and simply do what you told us from an Ehcache perspective.

However the call and the method (at least name) should be changed to reflect this better.